### ISSUE TITLE
Trying deploy to Heroku

### ISSUE THREAD

**Issue by Trentyn** [galnir/Master-Bot#480]
-----> Building on the Heroku-20 stack
-----> Determining which buildpack to use for this app
-----> Node.js app detected
       
-----> Creating runtime environment
       
       NPM_CONFIG_LOGLEVEL=error
       NODE_VERBOSE=false
       NODE_ENV=production
       NODE_MODULES_CACHE=true
       
-----> Installing binaries
       engines.node (package.json):  unspecified
       engines.npm (package.json):   unspecified (use default)
       
       Resolving node version 14.x...
       Downloading and installing node 14.16.0...
       Using default npm version: 6.14.11
       
-----> Installing dependencies
       Installing node modules
       npm ERR! Error while executing:
       npm ERR! /usr/bin/git ls-remote -h -t ssh://git@github.com/discordjs/Commando.git
       npm ERR! 
       npm ERR! Host key verification failed.
       npm ERR! fatal: Could not read from remote repository.
       npm ERR! 
       npm ERR! Please make sure you have the correct access rights
       npm ERR! and the repository exists.
       npm ERR! 
       npm ERR! exited with error code: 128
       
       npm ERR! A complete log of this run can be found in:
       npm ERR!     /tmp/npmcache.sN3uW/_logs/2021-03-25T10_28_47_027Z-debug.log
-----> Build failed
       
       We're sorry this build is failing! You can troubleshoot common issues here:
       https://devcenter.heroku.com/articles/troubleshooting-node-deploys
       
       Some possible problems:
       
       - Node version not specified in package.json
         https://devcenter.heroku.com/articles/nodejs-support#specifying-a-node-js-version
       
       Love,
       Heroku
       
 !     Push rejected, failed to compile Node.js app.
 !     Push failed

**Comment by galnir** [comment]
For some reason your heroku instance can't access one of the dependencies' repository. I never hosted the bot on heroku or others like repl.it, so maybe someone else will be able to solve your issue

**Comment by Bacon-Fixation** [comment]
First on github create a private Fork of Master-Bot and add a completed config.json file to your fork
Create a File called `Procfile` and add `worker: node index.js` and add it to your fork (`Master-Bot/` with the config.json file)

Go to Heroku.com and Create a New App
![image](https://user-images.githubusercontent.com/12632936/112651369-5289f980-8e1a-11eb-9186-119675b147db.png)

Name your app and click on Create App
![image](https://user-images.githubusercontent.com/12632936/112651626-95e46800-8e1a-11eb-9b8a-7f832fe578b3.png)

Go to Deployment Method and click connect to Github, log in and find the repo you created click on connect
![image](https://user-images.githubusercontent.com/12632936/112651962-e8258900-8e1a-11eb-83e5-be7bb730d0ed.png)

scroll down to Manual Deploy section and click deploy branch
![image](https://user-images.githubusercontent.com/12632936/112652221-291d9d80-8e1b-11eb-955d-4fabeee6427b.png)

this will start the Install for the Dependencies (wait for it to complete...)
once the install is complete go to the Resources Tab and click the pencils
![image](https://user-images.githubusercontent.com/12632936/112653813-caf1ba00-8e1c-11eb-92e0-7af3db8e44ee.png)

Disable `Web` then Enable `Worker` click confirm on both
![image](https://user-images.githubusercontent.com/12632936/112654008-f5437780-8e1c-11eb-8c25-e222d234c16e.png)

now lets see if it is running in the top right click on more > view log
![image](https://user-images.githubusercontent.com/12632936/112654726-b104a700-8e1d-11eb-9317-a5098f753774.png)

if it is working show look like this
![image](https://user-images.githubusercontent.com/12632936/112654910-e3160900-8e1d-11eb-8fb2-247d299aa136.png)

Much Love
-Bacon

**Comment by Bacon-Fixation** [comment]
@Trentyn, just wanted to get back with you to see if you were still having issues.

Much Love
-Bacon

**Comment by nettlees** [comment]
bacon you are a legend!

**Comment by Natemo6348** [comment]
just wanted to say that this also happens on replit sometimes, haven't tested on my pc if it happens as well, but usually installing the module individually and not at the same time as the rest usually helps, idk what to do for heroku

**Comment by galnir** [comment]
Bacon is indeed a legend. @Bacon-Fixation would you mind copying your comment to a new issue so I can pin it?

**Comment by Bacon-Fixation** [comment]
done @galnir #490 :D

Much Love
-Bacon
